developer/src/server/src/site/chargrid.js

```diff
@@ -73,17 +73,10 @@
   function updateLogCursor() {
     var i, selStart, selLength, selDirection;
 
-    if(keyman.isPositionSynthesized()) { // this is an internal function
-      // For touch devices, we need to ask KMW
-      selStart = 0;
-      selLength = 0;
-      selDirection = 'forward';
-    } else {
-      // For desktop devices, we use the position reported by the textarea control
-      selStart = ta1.selectionStart;
-      selLength = ta1.selectionEnd - ta1.selectionStart;
-      selDirection = ta1.selectionDirection;
-    }
+    // We use the position reported by the textarea control
```

Comment:
We'll need to test these changes on mobile devices as well as mobile-on-desktop mode, more in terms of making sure Developer test still works as expected without the touch alias stuff (I mean, it should, but focus changes could be relevant)

Reply:
Would a simple, naive "try it from your phone on the same network" test based on one of the URLs put forth by Developer be enough, or is there more detail to consider when writing such a test?

Reply:
Yes, can even test in Chrome's mobile device emulation.

Reply:
Documented as a TODO item on the main feature branch PR (#8560).